modules/ztp-pre-staging-downloading-artifacts.adoc

```diff
@@ -0,0 +1,367 @@
+
+.Prerequisites
+
+* You partitioned the storage device.
+* You have enough space for the images on the partitioned device.
+* You connected the bare-metal server to the Internet.
+* You have a valid pull secret.
+
+.Procedure
+
+. Check the {rh-rhacm} and Multicluster engine (MCE) version by running the following commands in the hub cluster:
++
+[source,terminal]
+----
+$ oc get csv -A | grep -i advanced-cluster-management
+----
+
++
+.Example output for {rh-rhacm} version
+[source,terminal]
+----
+open-cluster-management                            advanced-cluster-management.v2.5.4           Advanced Cluster Management for Kubernetes   2.5.4                 advanced-cluster-management.v2.5.3                Succeeded
+----
+
++
+[source,terminal]
+----
+$ oc get csv -A | grep -i multicluster-engine
+----
+
++
+.Example output for MCE version
+[source,terminal]
+----
+multicluster-engine                                cluster-group-upgrades-operator.v0.0.3       cluster-group-upgrades-operator              0.0.3                                                                   Pending
+multicluster-engine                                multicluster-engine.v2.0.4                   multicluster engine for Kubernetes           2.0.4                 multicluster-engine.v2.0.3                        Succeeded
+multicluster-engine                                openshift-gitops-operator.v1.5.7             Red Hat OpenShift GitOps                     1.5.7                 openshift-gitops-operator.v1.5.6-0.1664915551.p   Succeeded
+multicluster-engine                                openshift-pipelines-operator-rh.v1.6.4       Red Hat OpenShift Pipelines                  1.6.4                 openshift-pipelines-operator-rh.v1.6.3            Succeeded
+----
+
+. Query the AI version on your hub cluster:
++
+[source,terminal]
+----
+$ oc get cm assisted-service -n open-cluster-management -oyaml | \ <1>
+grep -E "AGENT_DOCKER_IMAGE|CONTROLLER_IMAGE|INSTALLER_IMAGE"
+----
+<1> The namespace can differ based on the installed {rh-rhacm} version.
+
+. Save the output, which is required for downloading the artifacts.
+
++
+.Example output for AI version
+[source,terminal]
+----
+AGENT_DOCKER_IMAGE: registry.redhat.io/multicluster-engine/assisted-installer-agent-rhel8@sha256:da1753f9fcb9e229d0a68de03fac90d15023e647a8db531ae489eb93845d5306
+CONTROLLER_IMAGE: registry.redhat.io/multicluster-engine/assisted-installer-reporter-rhel8@sha256:e8d6b78248352b1a8e05a22308185a468d4a139682d997a7f968b329abbc02cd
+INSTALLER_IMAGE: registry.redhat.io/multicluster-engine/assisted-installer-rhel8@sha256:33abd6e21cfdc36dd4337fa6f3c3442d33fc3f976471614dca5b8ef749e7a027
+----
+
+. To access the container registry, copy a valid pull secret on the server to be installed:
+
+.. Create the `.docker` folder:
++
+[source,terminal]
+----
+$ mkdir /root/.docker
+----
+
+.. Copy the `config.json` file to the previously created `.docker` folder:
```

Comment:
Copy the valid pull secret named 'config.json' to the... or renamed to... 

I mean, a valid pull secret to pull the images in the usual json format. The point is that it needs to be in the /root/.docker folder and named as config.json so it is the default path to be used by the container tools in the server or inside the container since it is mounted (-v /root/.docker).